src/main/java/com/google/sps/servlets/WaterPollutionServlet.java

```diff
@@ -59,21 +71,57 @@
 
     public static final int DEFAULT_SCORE = 100;
 
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
     @Override
     public void init() {
     }
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         response.setContentType("application/json");
+        Entity entity;
+        try {
+			entity = datastore.get(getKeyFromLocality(request.getParameter(TOWN_PARAMATER), request.getParameter(STATE_PARAMATER)));
```

Comment:
Formatting error here?

Reply:
That's strange, it doesn't look like that on my editor